xml/hardening.xml

```diff
@@ -4074,9 +4104,9 @@ RhostsRSAAuthentication no</screen>
       <literal>fs.protected_symlinks</literal> or their corresponding
       <filename>/proc</filename> entries:
      </para>
-<screen>$ cat /proc/sys/fs/protected_hardlinks
+<screen>&prompt.user; cat /proc/sys/fs/protected_hardlinks
```

Comment:
Two issues:

* Remove the space between `&prompt.user;` and `cat`.
* Wrap `cat` with `<command>`.